foqus_lib/foqus.py

```diff
@@ -93,7 +51,7 @@ def guiImport():
             QFileDialog
         from PyQt5.QtGui import QPixmap, QPainter
         from PyQt5 import QtSvg
-        #import foqus_lib.gui.icons_rc
+        import foqus_lib.gui.icons_rc
```

Comment:
I don't see any reason for this import other than to see if it works.  Could remove this, but it isn't hurting anything.

Reply:
If it really isn't needed, I vote for removing it.  It could come to bite us someday.  Try running pylint on it for even more fun...